src/community/advanced-data-card.tsx

```diff
@@ -282,65 +284,69 @@ export const AdvancedDataCard = React.forwardRef<HTMLDivElement, AdvancedDataCar
+                                className={classNames(
+                                    styles.cardContentStyle,
+                                    !hasFooter && !hasExtras ? styles.minHeight : ''
+                                )}
+                            >
+                                <Box paddingTop={8}>
+                                    <Inline space={0}>
+                                        <Stack space={8}>
+                                            {stackingGroup}
+                                            <CardContent
+                                                headline={headline}
+                                                pretitle={pretitle}
+                                                pretitleAs={pretitleAs}
+                                                pretitleLinesMax={pretitleLinesMax}
+                                                title={title}
+                                                titleAs={titleAs}
+                                                titleLinesMax={titleLinesMax}
+                                                subtitle={subtitle}
+                                                subtitleLinesMax={subtitleLinesMax}
+                                                description={description}
+                                                descriptionLinesMax={descriptionLinesMax}
+                                            />
+                                        </Stack>
+                                        {/** Hack to avoid content from rendering on top of the top action buttons */}
+                                        {!stackingGroup && <div style={topActionsStylesWithoutIcon} />}
+                                    </Inline>
+                                </Box>
+                            </div>
+                            <div style={{flexGrow: 1}} />
+                            {hasExtras && (
+                                <Box paddingTop={16} paddingBottom={24} width="100%">
+                                    {extra.map((item, index) => {
+                                        return (
+                                            <div key={index}>
+                                                <div className={styles.paddingX}>{item}</div>
+
+                                                {index + 1 !== extra.length && (
+                                                    <Box paddingY={extraDividerPadding}>
+                                                        <Divider />
+                                                    </Box>
+                                                )}
+                                            </div>
+                                        );
+                                    })}
+                                </Box>
+                            )}
+                        </Touchable>
+                        <CardActionsGroup actions={actions} onClose={onClose} padding={7} />
```

Comment:
why 7? weird number

Reply:
Added a comment to explain why we used 7px in here.